fontforge/utanvec.c

```diff
@@ -34,7 +34,7 @@
 #include <math.h>
 #include <assert.h>
 
-#define UTMARGIN (1e-8)
+#define UTMARGIN (5e-7)
```

Comment:
Did you compute this somehow or is it just the result of testing against old and new targets?

Reply:
Actually this is just wrong -- so good catch.  What I intended was either 1e-7 or 5e-8. 

This value went through a lot of testing with many sources but one of the #4406 cases had a difference just slightly larger than 1e-8. (Scrolling back up the value was
`{x = 1, y = 1.1978936413822867e-08}`, which needed to match `{1,0}`.) So I first switched it to 1e-7 and then intended to move it back down to 5e-8 and futzed the edit. 

Which would you recommend, or is there some more principled way of determining? I suppose I could trace back through the execution patch to figure out where in particular the values are coming from ...

Reply:
Whatever is decided on, certainly there should be a comment explaining why all "random-seeming" constants were chosen. :-)

Reply:
I guess. Half the constants I've run into in Skia are either uncommented or say "I chose this at random". I can put "arrived at through testing" on each of them but I'm not sure what good it will do.

Anyway absent other feedback I'm going to go with 1e-7 for this one.